x11-drivers/nvidia-drivers/files/nvidia-430.conf

```diff
@@ -17,4 +17,4 @@ options nvidia \
 	NVreg_DeviceFileUID=0 \
 	NVreg_ModifyDeviceFiles=1 \
 	NVreg_PreserveVideoMemoryAllocations=1 \
-	NVreg_TemporaryFilePath=/tmp
+	NVreg_TemporaryFilePath=/var/tmp
```

Comment:
why was the temporary directory moved for it?

Reply:
It's used for hibernation storage, /tmp typically being tmpfs felt like an unreasonable default.